src/solvers/dgmulti/dg.jl

```diff
@@ -3,17 +3,21 @@
 # we need to opt-in explicitly.
 # See https://ranocha.de/blog/Optimizing_EC_Trixi for further details.
 @muladd begin
-
+#! format: noindent
 
 # out <- A*x
 mul_by!(A) = @inline (out, x) -> matmul!(out, A, x)
 mul_by!(A::T) where {T <: SimpleKronecker} = @inline (out, x) -> mul!(out, A, x)
 mul_by!(A::AbstractSparseMatrix) = @inline (out, x) -> mul!(out, A, x)
-mul_by!(A::LinearAlgebra.AdjOrTrans{T, S}) where {T, S<:AbstractSparseMatrix} = @inline (out, x) -> mul!(out, A, x)
+function mul_by!(A::LinearAlgebra.AdjOrTrans{T, S}) where {T, S <: AbstractSparseMatrix}
+    @inline (out, x) -> mul!(out, A, x)
+end
```

Comment:
Would prefer no function blocks here (and below) but it's not a big issue.

Reply:
As before - the a short function style is used but the line length is violated, this will be wrapped into a regular `function`